src/components/tasks/card/card.scss

```diff
@@ -0,0 +1,43 @@
+.card {
+  border: 0.2rem solid black;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  max-width: 25rem;
+  margin-bottom: 2rem;
```

Comment:
Minor: I would encourage you to use `em` and use a container font-size to control the elements.
Global change to root font-size can have some major design bugs.

Reply:
Done !